cypress/integration/samples.spec.js

```diff
@@ -133,6 +133,7 @@ describe('examples', () => {
     cy.get('#output').should("contain", 'Call | Received remote stream');
     cy.get('#output').should("contain", 'Call | ICE State : connected');
     cy.get('#output').should("contain", 'Call | ICE State : completed');
+    cy.wait(500);
```

Comment:
It might be better to use an approach like `waitUntil` instead of `.wait(500)` if possible.

Reply:
I removed that line and increased timeout for all DOM related steps of integration tests instead.